test/unit/user_test.rb

```diff
@@ -25,11 +25,11 @@
 
 class UserTest < ActiveSupport::TestCase
 
-  test "user visible" do
+  test "user index" do
```

Comment:
[pronto-rubocop warning] Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.